src/Nethermind/Nethermind.Runner/RunnerApp.cs

```diff
@@ -160,11 +143,47 @@ IConfigProvider BuildConfigProvider()
                     configFilePath = configPathVariable;
                 }
 
+                configFilePath = configFilePath.GetApplicationResourcePath();
+
+                if (!Path.HasExtension(configFilePath) && !configFilePath.Contains(Path.DirectorySeparatorChar))
```

Comment:
This looks like similar code as below (3x similar versions), maybe make a method?